### PR TITLE
[frontend] add share button in connectors card/slug page (private)

### PR DESCRIPTION
**Pull request by jpkha**
# Context
> Briefly describe the purpose and context of this PR. Include relevant screenshots or screen recordings for the product team.

## Related issues

- Related to #1058

# Checklist

## Quality

- [x] I consider this work complete and ready for review
- [x] I tested all features and edge cases
- [x] I refactored code where necessary to improve quality
- [x] I made sure my code meets development guidelines
- [x] I performed a self-review of my code

## Testing

- [ ] I wrote test cases for the relevant use cases (unit, integration or e2e tests)
- [ ] (Bug fixes only) I added tests that reproduce and verify the fix

### What tests have been made

- [x] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Local manual tests

# Additional information
> Any additional context, dependencies, or concerns reviewers should know about.

### CODE REVIEW

File: apps/portal-api/src/modules/services/services.resolver.ts [hunk 1]
```diff
@@ -46,6 +53,25 @@ import {
...
 const resolvers: Resolvers = {
   ServiceInstance: {
+    __resolveType(service_instance) {
+      const integrationFeedMapping = {
+        [INTEGRATION_FEED_CONNECTORS_TYPE]: 'Connector',
+        [INTEGRATION_FEED_CSV_FEED_TYPE]: 'CsvFeed',
+      };
+      const typeMapping = {
+        [OPENAEV_SCENARIO_DOCUMENT_TYPE]: 'OpenAEVScenario',
+        [CUSTOM_DASHBOARD_DOCUMENT_TYPE]: 'OpenCTICustomDashboard',
```
Comment (Kroustille) [inline apps/portal-api/src/modules/services/services.resolver.ts]:
```suggestion
        [OPENCTI_CUSTOM_DASHBOARD_DOCUMENT_TYPE]: 'OpenCTICustomDashboard',
```

**Review by Kroustille — APPROVED** [review]
